taipy/gui/_renderers/factory.py

```diff
@@ -597,6 +598,20 @@ class _Factory:
             ]
         )
         ._set_propagate(),
+        "Progress": lambda gui, control_type, attrs: _Builder(
```

Comment:
this should be `progress` and not `Progress`

Reply:
I've changed it to `progress` now. There's a different error this time:

`AssertionError: <CircularProgress not in <div class="md-para" key="div.0">
               <Progress key="Progress.0" libClassName="taipy-progress" value="&lt;Empty&gt;"></Progress>
               </div>`

I'll try to fix this ASAP. Also, if required, I'll ask here if I encounter any doubt or problem 👍

Reply:
`<CircularProgress` => `<Progress` in test

Reply:
Yes, now the test is passing 👍 Thanks

Reply:
@FredLL-Avaiga, actually the 2nd test which I've created: <br>
```
def test_circular_progress_determinate_md(gui: Gui, helpers):
    gui._bind_var_val("value", 50)
    md_string = '<|progress|value={value}|showValue=True|>'
    expected_list = ["<Progress", 'value="50"']
    helpers.test_control_md(gui, md_string, expected_list)
```

This test is failing this way, but when I'm doing it like this, it's passing:

```
def test_circular_progress_determinate_md(gui: Gui, helpers):
    #gui._bind_var_val("value", 50)
    md_string = '<|progress|value=50|showValue=True|>'
    expected_list = ["<Progress", 'value="50"']
    helpers.test_control_md(gui, md_string, expected_list)
```

Here's the error which I'm getting for the 1st one:
```
RuntimeError: Working outside of application context.
E
E           This typically means that you attempted to use functionality that needed
E           the current application. To solve this, set up an application context
E           with app.app_context(). See the documentation for more information.
```

Reply:
Could you please guide that why it's failing for the 1st one? I've also used the `_bind_var_val` function for the `value`.

Reply:
I've written only - `value=50` not `defaultValue={50}` and that's working